### ISSUE TITLE
Setting HTTP Auth requires restart, user isn't informed

### ISSUE THREAD

**Issue by ZJvandeWeg** [flowforge/flowforge#1355]
### Current Behavior

When a user sets up HTTP Auth it requires the runtime to be restarted before it takes effect. The user doesn't know however, and they should be.

### Environment

- FlowForge version: v1.1
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

**Comment by knolleary** [comment]
This is true for all settings, so the required UX improvement isn't limited to this one property.
